Automattic/jetpack
# Contact Form: Form Dropdown revamp

Fixes #27468

#### Changes proposed in this Pull Request:
* Introduces several changes to the Dropdown Field block aiming for better UI and UX

#### Other information:


#### Jetpack product discussion
pcsBup-r2-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a post and add a Form block
* Include a Dropdown Field to the Form and play with the block to ensure it's working properly and aligns with the discussion in the issue above
